ground/src/main/java/com/google/android/ground/model/geometry/GeometryValidator.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.ground.model.geometry
+
+object GeometryValidator {
+  private fun List<Any>.isFirstAndLastSame(): Boolean = firstOrNull() == lastOrNull()
```

Comment:
I don't think we need this for now. I'd just write the comparison directly in the impl of `isComplete`.

Reply:
Done